app/src/main/java/com/github/se/eduverse/ui/navigation/BottomNavigationMenu.kt

```diff
@@ -25,18 +30,27 @@ fun BottomNavigationMenu(
     tabList: List<TopLevelDestination>,
     selectedItem: String
 ) {
-  BottomNavigation(
+  val context = LocalContext.current
+  NavigationBar(
       modifier = Modifier.fillMaxWidth().height(60.dp).testTag("bottomNavigationMenu"),
-      backgroundColor = MaterialTheme.colorScheme.surface,
-      content = {
+      containerColor = Color(0xFFD4DEE8) // Background color for the NavigationBar
+      ) {
         tabList.forEach { tab ->
-          BottomNavigationItem(
+          NavigationBarItem(
               icon = { Icon(tab.icon, contentDescription = null) },
               label = { Text(tab.textId) },
-              selected = tab.route == selectedItem,
+              colors =
+                  NavigationBarItemColors(
+                      Color.Black,
+                      Color.Black,
+                      Color(0xFF37CED5),
```

Comment:
You could define val lightBlue = Color(0xFF37CED5) to replace the hex code. This will improve readability and might make color adjustments simpler in the future.